src/benchmarks/micro/corefx/System.Drawing/Perf_Image_Load.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Drawing.Imaging;
+using System.IO;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Extensions;
+using MicroBenchmarks;
+
+namespace System.Drawing.Tests
+{
+    [BenchmarkCategory(Categories.CoreFX)]
+    public class Perf_Image_Load
+    {
+        private static readonly ImageTestData[] TestCases = {
+            new ImageTestData(ImageFormat.Bmp),
+            new ImageTestData(ImageFormat.Jpeg),
+            new ImageTestData(ImageFormat.Png),
+            new ImageTestData(ImageFormat.Gif)
+        };
```

Comment:
the old benchmarks were most probably by accident using only `Bmp`, I have fixed that

https://github.com/dotnet/corefx/blob/9940166bff52155600f25f15da13a83c982526d2/src/System.Drawing.Common/tests/Performance/Perf_Image_Load.cs#L122